src/acl/external/AD_group/ext_ad_group_acl.cc

```diff
@@ -848,7 +845,7 @@ main(int argc, char *argv[])
         if ((p = strchr(buf, '\r')) != NULL)
             *p = '\0';      /* strip \r */
 
-        debug("Got '%s' from Squid (length: %d).\n", buf, strlen(buf));
+        debug("Got '%s' from Squid (length: %d).\n", buf, static_cast<int>(strlen(buf)));
```

Comment:
Please fix format %code instead of casting strlen() result to the wrong type. The fewer bad examples we leave in our code, the better. See also: Commit a20724e4.

Reply:
Done in commmit dec50ce03a